src/main/java/com/minecolonies/coremod/client/gui/ViewFilterableList.java

```diff
@@ -35,7 +35,7 @@
     /**
      * Description label Id.
      */
-    private static final String DESC_LABEL = "desc";
+    public static final String DESC_LABEL = "desc";
```

Comment:
If this needs to be public you are 100% doing it wrong.
This field describes an internal reference, and there should be never a need to make it public.
Expose the label via a method instead.